libs/product/driver/shopify/src/models/shopify-money.ts

```diff
@@ -0,0 +1,4 @@
+export interface ShopifyMoneyV2 {
+  amount: number;
+  currencyCode: string;
+}
```

Comment:
This model (and the associated factory) should be in `@daffodil/driver/shopify` as it is a generic model and doesn't have anything to do with products specifically. I really like how you are breaking up the types!